src/Collab.js

```diff
@@ -151,6 +153,13 @@ class Collab extends Component {
         });
         this.addNotificationAlert("Mode has been changed to: " + mode);
     }
+
+    handleLineWrap() {
+        this.setState({
+            options: {lineNumbers: true, mode: this.mode, lineWrapping: true}
+        });
+        this.addNotificationAlert("Line-Wrapping has been set to: "+ this.lineWrapping);
```

Comment:
This shouldn't be an alert -- also where is this.lineWrapping coming from?

Reply:
The alert was for debug, it will be gone in my next commit. This is in-progress code for word wrapping in codemirror. I opted to submit some of my changes so I could get feedback instead of spending 2-3 hours picking through my code to separate the wordWrap code from the homepage code. I'll create a separate branch for it once I fix the issues with the homepage.